docs/installation.md

```diff
@@ -219,6 +219,8 @@ with real users.
 | AWS_VPC_ID | aws, aws-single-node | The AWS VPC to use |
 | AWS_SSH_KEY_NAME | aws, aws-single-node | The name of an SSH key in the AWS region to use for ansible |
 | GCP_PROJECT | gcp | The GCP project id to deploy in |
+| GCP_SSH_KEY | gcp | The content of an ssh public key file to use for authentication (optional, only required if not using gcloud). |
```

Comment:
This is used as a filename not the "content"?

Sorry for being retentive.

Reply:
Done.